intl/msg_hash_nl.c

```diff
@@ -266,15 +266,15 @@ const char *msg_hash_to_str_nl(enum msg_hash_enums msg)
       case MENU_ENUM_LABEL_VALUE_FRONTEND_COUNTERS:
          return "Frontend Prestatie Tellers";
       case MENU_ENUM_LABEL_VALUE_HELP:
-         return "Help";
+         return "RetroArch Tour";
```

Comment:
Won't that be a good thing?

>Lakka is the official Linux distribution of RetroArch and the libretro ecosystem.
>
>Each game system is implemented as a libretro core, while the frontend RetroArch takes care of inputs and display. This clear separation ensures modularity and centralized configuration.

Got anything else (longer) than "Tour" in mind? 
```
Information
Configurations
Tour
Quit RetroArch
```